.pylintrc

```diff
@@ -20,4 +20,5 @@ disable=
     too-many-ancestors,
     missing-module-docstring,
     too-many-arguments,
-    too-many-branches
\ No newline at end of file
+    too-many-branches,
+    too-many-return-statements
```

Comment:
Why are we adding this in here? 

If its a single line/case, maybe just add a:
```
# pylint: disable = too-many-return-statements
```

Reply:
Okei, lets make it general then! 
I'll remove the other places that have this individually!